<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>代码生成器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <widget class="QWidget" name="page_13"/>
        <widget class="QWidget" name="page_14"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>650</height>
         </size>
        </property>
        <widget class="QWidget" name="page_1">
         <widget class="QScrollArea" name="scrollArea_1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>353</x>
              <y>50</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是数据库配置页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QScrollArea" name="scrollArea_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>450</x>
              <y>80</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是配置确认配置页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QScrollArea" name="scrollArea_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_9">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>380</x>
              <y>160</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是代码生成页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QScrollArea" name="scrollArea_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>350</x>
              <y>70</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QStackedWidget" name="stackedWidget_3">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>0</y>
              <width>711</width>
              <height>641</height>
             </rect>
            </property>
            <widget class="QWidget" name="page">
             <widget class="QWidget" name="verticalLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>10</y>
                <width>411</width>
                <height>131</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>视图准备</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>请选择用于生成代码的视图，并为每张表进行相应配置。</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="page_6">
             <widget class="QWidget" name="verticalLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>711</width>
                <height>641</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QScrollArea" name="scrollArea_6">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_4">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>707</width>
                    <height>637</height>
                   </rect>
                  </property>
                  <widget class="QLabel" name="label_view">
                   <property name="geometry">
                    <rect>
                     <x>30</x>
                     <y>30</y>
                     <width>201</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                  <widget class="QLabel" name="label_9">
                   <property name="geometry">
                    <rect>
                     <x>30</x>
                     <y>90</y>
                     <width>121</width>
                     <height>16</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>接口查询字段选择</string>
                   </property>
                  </widget>
                  <widget class="QTableWidget" name="tableWidget">
                   <property name="geometry">
                    <rect>
                     <x>30</x>
                     <y>130</y>
                     <width>651</width>
                     <height>271</height>
                    </rect>
                   </property>
                  </widget>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>281</width>
              <height>641</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_left_2">
             <item>
              <widget class="QScrollArea" name="scrollArea_left_2">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_left_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>277</width>
                  <height>637</height>
                 </rect>
                </property>
                <widget class="QWidget" name="horizontalLayoutWidget">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>281</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_all_2" stretch="1,16">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="checkBox_all_2">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_all_2">
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	
}</string>
                    </property>
                    <property name="text">
                     <string>PushButton</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>991</width>
            <height>641</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>989</width>
             <height>639</height>
            </rect>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>390</x>
              <y>70</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>这是数据库表配置页面</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_last">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>上一步</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_next">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>下一步</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
